delegate-app/Main.hs

```diff
@@ -349,7 +351,13 @@ main :: IO ()
 main = do
   port <- lookupEnv "PORT"
   -- FIXUP: parse actual adress and other params
-  hydraNodeNumber <- read . fromJust <$> lookupEnv "HYDRA_NODE_NUMBER"
+  envHydraNode <- lookupEnv "HYDRA_NODE_NUMBER"
```

Comment:
There is FIXUP and PR todo which state that this is temporarily solution. It should be changed into proper CLI parser, not added errors.